db/Dockerfile

```diff
@@ -19,11 +19,11 @@ RUN apk add --no-cache \
 COPY setup-wale.sh /docker-entrypoint-initdb.d/
 
 #Healthcheck to make sure container is ready
-HEALTHCHECK CMD pg_isready || exit 1
+HEALTHCHECK CMD pg_isready -U postgres || exit 1
```

Comment:
I remove the user in PR #318 because the user is not always postgres. So the healthcheck return a FATAL log message.